### ISSUE TITLE
SignUp and Login

### ISSUE THREAD

**Issue by coderchira** [CrackDataPots/trainingsurveyapp#6]
A user should be able to SignUp and Login into the application

**Comment by kneerunjun** [comment]
Do we prefer the user logging in with separate credentials or we want the user to login using his Google / Facebook credentials ?

**Comment by coderchira** [comment]
Good spotting! We will provide both the options. I will include this in the description, right away!

**Comment by kneerunjun** [comment]
So no need to maintain any user profiles on the app base

**Comment by coderchira** [comment]
No, We could still have user profiles maintained separately for the app .
For eg. A user can have a unique 'UserName' and a 'Password' just for the App.

And one more separate case can be where the user logs in by using an organization's credentials.

**Comment by kneerunjun** [comment]
Not a big fan of keeping double authentication data ..

**Comment by coderchira** [comment]
Well, I now see the point clear enough! So, we will go ahead with only the Google/FB authentication. :+1:
